newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -37,6 +43,9 @@ export default class HighlightedInstance {
   }
 
   setInstance(instance: gdInitialInstance | null) {
+    this.isHighlightedInstanceOf3DObject = instance
+      ? this.isInstanceOf3DObject(instance)
```

Comment:
Not a big fan of having to drill this function isInstanceOf3DObject, I guess at some point this could be saved on the instance itself in memory.
But I guess it works and it's only for a single instance, so not sure expensive to run anyway.